Source/WebKit/WebProcess/cocoa/WebProcessCocoa.mm

```diff
@@ -209,6 +209,15 @@
 
 id WebProcess::accessibilityFocusedUIElement()
 {
+    auto retrieveFocusedUIElementFromMainThread = [] () {
+        return Accessibility::retrieveAutoreleasedValueFromMainThread<id>([] () -> RetainPtr<id> {
+            WebPage* page = WebProcess::singleton().focusedWebPage();
```

Comment:
We should store `page` using a smart pointer, probably either `CheckedPtr` or `RefPtr`.